### ISSUE TITLE
Astropy Conflict

### ISSUE THREAD

**Issue by kaya4me2** [NASA-Planetary-Science/sbpy#313]
**High-level problem description**
astropy version  conflict 

**What did you do?**
Installing collected packages: PyYAML, pyerfa, astropy, photutils
  Attempting uninstall: astropy
    Found existing installation: astropy 3.2.3
    Uninstalling astropy-3.2.3:
      Successfully uninstalled astropy-3.2.3
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
sbpy 0.2.2 requires astropy<4.0,>=3.0, but you have astropy 5.0 which is incompatible.
Successfully installed PyYAML-6.0 astropy-5.0 photutils-1.3.0 pyerfa-2.0.0.1
(sbpy) bash-3.2$ 

**What did you expect?**
Resolved dependencies 

**What did really happen?**
Failed install ? 

**Provide information on your environment:**
MacOSX Astronconda 
operating system and version: [Linux (which distribution?), MacOS, Win]
sbpy version: 
astropy version:
numpy version:

**Comment by mkelley** [comment]
Hi @kaya4me2, thanks for the issue, but this report is a little bit unclear to me.  Were you installing sbpy or upgrading astropy, or else what command brought you to this error?

**Comment by kaya4me2** [comment]
created new environment (condo create --name spy python 3.8), 
then pip installed sbpy , 
then attempted to get photutils into the environment as one of the user add-on, like ginga or pyradex

re:

The following packages will have to be installed manually, if the user wants to use them:

oorb: For orbit transformations (oo_transform) and propagations (oo_propagate), as well as ephemerides calculations (from_oo).
pyradex: For non-LTE production rate calculation related to cometary activity (NonLTE).
ginga and photutils: To interactively enhance images of comets with the CometaryEnhancement Ginga plugin.

**Comment by mkelley** [comment]
Ah, I think this is a version conflict between sbpy 0.2.2 and photutils 1.3.0.  Unfortunately, this means using an old version of photutils or using an advanced version of sbpy.  If your goal is to use the Ginga plugin, it would be better to use an older photutils.  Looking at photutils's history, it seems sbpy 0.2.2 and photutils<1.0 can work together.  I tried it out in a virtual environment with success.  For reference here is what I did:

```
python3 -m venv sbpy
source sbpy/bin/activate
pip install -U pip setuptools wheel
pip install sbpy
pip install 'photutils<1.0'
pip install ginga[gtk3] scipy
```
